app/utils/transactionHelpers.ts

```diff
@@ -592,13 +613,36 @@ export function validateBakerStake(
     return undefined;
 }
 
+export function validateMemo(memo: string): string | undefined {
+    const asNumber = Number(memo);
+    if (
+        Number.isInteger(asNumber) &&
+        (asNumber > Number.MAX_SAFE_INTEGER ||
+            asNumber < Number.MIN_SAFE_INTEGER)
+    ) {
+        return `Only numbers up to ${Number.MAX_SAFE_INTEGER} is supported.`;
```

Comment:
I suggest replacing the above with:
```
Numbers greater than 9007199254740991 or smaller than -9007199254740991 are not supported.
```
If there is a space concern and you want to keep the current, then you need to change _is_ to _are_.